administering-tugboat-crew/index.md

```diff
@@ -0,0 +1,129 @@
+2. Click the **My Projects** link;
+3. Select the project where you want to add the user;
+4. Click the **Project Settings** link to the right of the project's title;
+5. In the **Invite a User to This Project** section, add the recipient's email
+   address, and select the appropriate
+   [user type](#user-permission-levels-explained) from the drop-down;
+6. Press the big blue **Invite** button!
+
+The user you've invited will get an email from `support@tugboat.qa` with a link
+to accept the invitation. If the user doesn't see the invite:
+
+- Have them check Inboxes and Spam for this email address, or;
+- **Copy Link** and share it another way from the Pending Invites section of the
+  Project Settings, or;
+- Hit the **Re-send** link from the Pending Invites section of the Project
+  Settings.
+
+![Copy Link or Re-Send from Pending Invites](_images/add_user_pending_invite.png)
+
+## Remove a user from a project
+
+1. Click your username in the upper right-hand corner, and go to Projects;
+2. Click the **My Projects** link;
+3. Select the project where you want to remove the user;
+4. Click the **Project Settings** link to the right of the project's title;
+5. In the **Manage Users** section, look for the user you want to remove, and
+   click the Remove link.
+
+## Change user permissions
+
+1. Click your username in the upper right-hand corner, and go to Projects;
+2. Click the **My Projects** link;
+3. Select the project where you want to remove the user;
+4. Click the **Project Settings** link to the right of the project's title;
+5. In the **Manage Users** section, look for the user whose permissions you want
+   to change, and select the appropriate
+   [user type](#user-permission-levels-explained) from the **Access** drop-down.
+
+## Add a Tugboat bot to your team
+
+By default, comments that Tugboat posts to the linked git provider account
+display as the user whose credentials link the account to Tugboat. If you
+instead want those comments to show as coming from Tugboat, you can add a
+Tugboat bot to your team:
+
+1. Create an account called "Tugboat Bot" at your preferred git provider; i.e.
+   GitHub, GitLab, BitBucket;
+2. Optional:
+   [Download the tugboat avatar](https://dashboard.tugboat.qa/static/Tugboat_AvatarLarge.zip)
+   to use for your Tugboat Bot account;
+3. Click your username in the upper right-hand corner, and go to Projects;
+4. Click the **My Projects** link;
+5. Select the project where you want to switch to the Tugboat bot;
+6. Click the **Repository Settings** link next to the repo where you want to
+   switch to the Tugboat bot;
+7. Scroll down to the **Provider Comments** section (GitHub Comments, GitLab
+   Comments or BitBucket Comments);
+8. Press the big blue **Change** button;
+9. Enter the authentication details for the Tugboat bot user you created at the
+   git provider;
+10. Specify permissions, or accept the default permissions, and press the **OK**
```

Comment:
These permissions are not configurable. They are a screenshot of the permissions required at the provider for the bot account you want to use for the comments.